kaal-domain/src/main/kotlin/cz/eman/kaal/domain/result/AdditionalErrorCode.kt

```diff
@@ -0,0 +1,40 @@
+package cz.eman.kaal.domain.result
+
+/**
+ * The enum defines additional error codes. Contains 3xx HTTP codes since some libraries may consider them as an error.
+ * Other additional codes are non-HTTP codes specific to this library (ex: [UNKNOWN_HOST]).
+ *
+ * The types of error:
+ * - 3xx: Redirects - Can be considered as error = not success, some libraries (ex.: Retrofit2 does not conciser 300 as
+ *   success).
+ * - Additional - Custom non http codes.
+ *
+ * @author eMan a.s.
+ * @since 0.9.0
+ */
+@Suppress("unused")
+enum class AdditionalErrorCode(override val value: Int) : ErrorCode {
```

Comment:
`AdditionalErrorCode` is very common name. I think that e.g. `NetworkErrorCode` is much better.